ui/trip_details/OBATripScheduleMapViewController.m

```diff
@@ -61,29 +61,37 @@ - (void)viewWillAppear:(BOOL)animated {
 
     if (_tripInstance && !_tripDetails) {
         @weakify(self);
-        _request = [_appDelegate.modelService
+        _request = [[OBAApplication sharedApplication].modelService
                     requestTripDetailsForTripInstance:_tripInstance
                                       completionBlock:^(id responseData, NSUInteger responseCode, NSError *error) {
                                           @strongify(self);
+
                                           if (error || responseCode >= 300) {
-                                              [self updateProgressViewWithError:error responseCode:responseCode];
+                                          [self         updateProgressViewWithError:error
+                                             responseCode:responseCode];
                                           }
                                           else {
-                                              OBAEntryWithReferencesV2 *entry = responseData;
-                                              self.tripDetails = entry.entry;
-                                              [self handleTripDetails];
+                                          OBAEntryWithReferencesV2 *entry = responseData;
+                                          self.tripDetails = entry.entry;
+                                          [self handleTripDetails];
                                           }
                                       }
 
-                                      progressBlock:^(CGFloat progress) {
-                                          @strongify(self);
-                                          if (progress > 1.0) {
-                                              [self.progressView setMessage:NSLocalizedString(@"Downloading...", @"message") inProgress:YES progress:progress];
-                                          }
-                                          else {
-                                              [self.progressView setInProgress:YES progress:progress];
-                                          }
-                                      }];
+                                        progressBlock:^(CGFloat progress) {
+                                            @strongify(self);
+
+                                            if (progress > 1.0) {
```

Comment:
It's out of scope for this pull request, but the uncrustify config needs to be fixed to prevent stupid stuff like this. A dedication to a particular line length at the expense of legibility doesn't help us.